api/query/atoms_test.go

```diff
@@ -616,6 +673,43 @@ func TestStructuredQuery_bindCount(t *testing.T) {
 	assert.Equal(t, expected, q.BindToRuns(runs...))
 }
 
+func TestStructuredQuery_bindLink(t *testing.T) {
+	e := shared.ParseProductSpecUnsafe("chrome")
+	f := shared.ParseProductSpecUnsafe("safari")
+	q := AbstractLink{
+		Pattern: "chromium.bug.com/abc",
+	}
+
+	runs := shared.TestRuns{}
+	runs = shared.TestRuns{
+		{
+			ID:                int64(0),
+			ProductAtRevision: e.ProductAtRevision,
+		},
+		{
+			ID:                int64(1),
+			ProductAtRevision: f.ProductAtRevision,
+		},
+	}
+
+	handler := func(w http.ResponseWriter, r *http.Request) {
+		http.ServeFile(w, r, "../../shared/metadata_testdata/gzip_testfile.tar.gz")
+	}
+	server := httptest.NewServer(http.HandlerFunc(handler))
+	defer server.Close()
+
+	shared.MetadataTestingURL = server.URL
```

Comment:
Does this really work? I don't see `MetadataTestingURL` ever used in any code paths.

I feel like this test *coincidentally* passed because your dummy test data happens to be the same as the current tip of tree of `wpt-metadata`.

A great way to make sure test data actually works is to put non-existent stuff in it (e.g. `foobarbaz`).